mfr/core.py

```diff
@@ -338,7 +338,10 @@ def copy_dir(src, dest):
     except shutil.Error as err:
         logger.warn(err)
     except OSError as err:
-        logger.debug('Skipping {src} (already exists)'.format(src=src))
+        if os.path.exists(src):
```

Comment:
Instead of using `os.path.exists` (which is prone to [race conditions](https://stackoverflow.com/questions/14574518/how-does-using-the-try-statement-avoid-a-race-condition), can you inspect some property of `err`?